example_pick.toml

```diff
@@ -0,0 +1,16 @@
+# An example of an optional config file for pick.
+# If no config file is available, sensible defaults will be used.
+# If you wish to use a config file, it must be located at ~/.pick/pick.toml
+
+# The encryption section controls the configuration of the safe encryption.
+[encryption]
+type = "aes"
```

Comment:
This should be renamed to `type = "aes-openpgp"` in case we later add other AES implementations (e.g. the native Go AES-GCM).